akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMultiMap.scala

```diff
@@ -115,10 +133,10 @@ final class ORMultiMap[A, B] private[akka] (private[akka] val underlying: ORMap[
    * INTERNAL API
    */
   @InternalApi private[akka] def put(node: UniqueAddress, key: A, value: Set[B]): ORMultiMap[A, B] = {
-    val newUnderlying = underlying.updated(node, key, ORSet.empty[B]) { existing ⇒
+    val newUnderlying = underlying.updated(node, key, ORSet.empty[B], valueDeltas = true) { existing ⇒
```

Comment:
I have just noticed this, and will fix it together with "small things" -> `valueDeltas` should depend on `mapTag == ORMultiMap.DeltaDelta... Tag`